python/ffsim/operators/fermi_hubbard.py

```diff
@@ -0,0 +1,79 @@
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Fermi-Hubbard operator."""
+
+from ffsim._lib import FermionOperator
+from ffsim.operators.fermion_action import cre_a, cre_b, des_a, des_b
+
+
+def fermi_hubbard(
+    norb: int,
+    t: float = 1,
+    U: float = 0,
+    mu: float = 0,
+    V: float = 0,
+    PBC: bool = False,
```

Comment:
Instead of single-letter name or acronyms, give these variables descriptive names.

Reply:
The new names still aren't descriptive enough. Suggestions:
- Instead of `t_hop`, `tunneling`
- Instead of `U_int`, `interaction`
- Instead of `mu_pot`, `chemical_potential`
- Instead of `V_pot`, `nearest_neighbor_interaction`
- Instead of `PBC`, `periodic`

You can also look at the corresponding function from [OpenFermion](https://github.com/quantumlib/OpenFermion/blob/e2fe1e678d973303fab231ce9b3cd83757413b94/src/openfermion/hamiltonians/hubbard.py#L20) for inspiration.

Also, let's not set default values for `t` and `U`.